src/types/recipe.ts

```diff
@@ -0,0 +1,55 @@
+export type Nutrient = {
+  name: string;
+  amount: number;
+  unit: string;
+};
+
+export type Recipe = {
+  id: number;
+  title: string;
+  image: string;
+  readyInMinutes: number;
+  nutrition?: {
+    nutrients: Array<{
+      name: string;
+      amount: number;
+      unit: string;
+    }>;
```

Comment:
[+] раз соответствующий тип уже определен выше, можно и здесь заменить:
```suggestion
    nutrients: Nutrient[];
```